data/gunicorn.conf.py

```diff
@@ -1,6 +1,10 @@
 # For more settings, see: https://docs.gunicorn.org/en/stable/settings.html
 
+import multiprocessing
+
 workers = 1
 worker_connections = 100
 bind = ":8050"
 timeout = 30
+worker_class = "gthread"
```

Comment:
```suggestion
# Worker is changed to prevent worker timeouts
# See: https://github.com/benoitc/gunicorn/issues/1801#issuecomment-585886471
worker_class = "gthread"
```

Reply:
The link in the comment is quite helpful. I'll commit the suggestion.